jenkins.sh

```diff
@@ -1,10 +0,0 @@
-#!/bin/bash -x
-set -e
-rm -f Gemfile.lock
-bundle install --path "${HOME}/bundles/${JOB_NAME}"
-
-bundle exec rake
-
-if [[ -n "$PUBLISH_GEM" ]]; then
-  bundle exec rake publish_gem
```

Comment:
You can also remove the `gem_publisher` dependency.

Reply:
Thanks 👍